readthedocs/rtd_tests/utils.py

```diff
@@ -106,6 +106,39 @@ def create_git_tag(directory, tag, annotated=False):
     chdir(path)
 
 
+def delete_git_tag(directory, tag):
+    env = environ.copy()
+    env['GIT_DIR'] = pjoin(directory, '.git')
+    path = getcwd()
+    chdir(directory)
+
+    command = ['git', 'tag', '-delete', tag]
+    check_output(command, env=env)
+    chdir(path)
```

Comment:
Instead of saving the path and recovering it before the functions returns, we should use this decorator: https://github.com/rtfd/readthedocs.org/blob/d804b533849ec0378c29496d93ff0eabbecb5048/readthedocs/doc_builder/base.py#L16-L27